medusa/providers/torrent/html/freshontv.py

```diff
@@ -58,7 +58,8 @@ def __init__(self):
         }
 
         # Proper Strings
-        self.proper_strings = ['PROPER', 'REPACK', 'REAL']
+        # Provider always returns propers and non-propers in a show search
+        self.proper_strings = ['']
```

Comment:
There is a loop on all the items in the list when doing a proper search, no items to loop for there won't be a search.